libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationView.java

```diff
@@ -346,6 +349,14 @@ public void updateWayNameView(@NonNull String wayName) {
     wayNameView.updateWayNameText(wayName);
   }
 
+  /**
+   * Updates the text of the speed limit view
+   * @param speedLimit to update the view
+   */
+  @Override public void updateSpeed(SpeedLimit speedLimit) {
```

Comment:
Maybe? 🤔 

```suggestion
  @Override public void updateSpeed(SpeedLimit limit) {
```

If not, for consistency 🤔 

```suggestion
  @Override public void updateSpeedLimit(SpeedLimit speedLimit) {
```